packages/atlas/src/views/viewer/AuctionView/AuctionView.tsx

```diff
@@ -0,0 +1,339 @@
+                          <Text variant="h300" secondary spacing={{ bottom: 2 }}>
+                            Current bid
+                          </Text>
+                          <FlexWrapper>
+                            <CurrentBidAvatar size="bid" />
+                            <BidderName variant="h400">{BID.bidder.name}</BidderName>
+                          </FlexWrapper>
+                        </ActionBarCell>
+                        <ActionBarCell>
+                          <Text variant="h300" secondary spacing={{ bottom: 2 }}>
+                            Bid amount
+                          </Text>
+                          <FlexWrapper>
+                            <CurrentBidJoyToken size={24} variant="silver" />
+                            <Text variant="h400">{BID.amount}</Text>
+                          </FlexWrapper>
+                        </ActionBarCell>
+                      </ActiveBidWrapper>
+                    ) : (
+                      <ActiveBidWrapper>
+                        <ActionBarCell>
+                          <Text variant="h300" secondary spacing={{ bottom: 2 }}>
+                            Current bid
+                          </Text>
+                          <Text variant="h400">Nobody has bid yet</Text>
+                        </ActionBarCell>
+                      </ActiveBidWrapper>
+                    )}
+                  </CurrentBidWrapper>
+                  <MinimumBidWrapper>
+                    <MinimumBid>
+                      <Text variant="h300" secondary>
+                        Minimum bid:
+                      </Text>
+                      <JoyTokenIcon variant="silver" size={24} /> <Text variant="h400">{MINIMUM_BID}</Text>
+                    </MinimumBid>
+                    <div>
+                      <Text variant="t100" secondary>
+                        Buy now: {FIXED_PRICE} tJOY
+                      </Text>
+                    </div>
+                  </MinimumBidWrapper>
+                  <TextField
+                    placeholder={`Min. ${MINIMUM_BID} tJOY`}
+                    nodeStart={<JoyTokenIcon variant="silver" size={24} />}
+                    nodeEnd={!!placedBid && <Pill variant="overlay" label={`$${convertToUSD(placedBid)}`} />}
+                    onChange={onPlaceBid}
+                    type="number"
+                    error={bidError || insufficientFoundsError}
+                    helperText={
+                      bidError
+                        ? 'Your bid must be higher than minimum bid'
+                        : insufficientFoundsError
+                        ? 'Insufficient funds.'
+                        : undefined
+                    }
+                  />
+                  {!!placedBid && placedBid > FIXED_PRICE && (
+                    <Text variant="t100" spacing={{ top: 2 }}>
+                      Max bid cannot be more than buy now price. Bidding for amount higher than Buy now will
```

Comment:
This doesn't fully follow the design, please check this user story: https://www.figma.com/file/wjeAOZq77pSUFZsYXJqKIB/NFT-Purchase-view?node-id=985%3A96806